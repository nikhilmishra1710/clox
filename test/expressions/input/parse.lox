// Note: This is just for the expression parsing chapter which prints the AST.
// expect: (+ (group (- 5.0 (group (- 3.0 1.0)))) (- 1.0))
// But since CLOX doesn't generate AST. A simple expression printer
print (5 - (3 - 1)) + -1;
print true;
print false;
print nil;
print "Str1" + " Str2";