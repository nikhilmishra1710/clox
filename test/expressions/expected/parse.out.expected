2
true
false
nil
Str1 Str2
